Model/CreditCardPaymentMethod.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Model;
+
+use OxidEsales\Eshop\Application\Model\Payment;
+use OxidEsales\Eshop\Core\Config;
+use OxidEsales\Eshop\Core\Registry;
+use Wirecard\Oxid\Core\Helper;
+
+use Wirecard\PaymentSdk\Config\Config as Wirecard_Config;
+use Wirecard\PaymentSdk\Config\CreditCardConfig;
+use Wirecard\PaymentSdk\Entity\Amount;
+use Wirecard\PaymentSdk\Transaction\CreditCardTransaction;
+use Wirecard\PaymentSdk\Transaction\Transaction;
+
+/**
+ * Class CreditCardPaymentMethod
+ *
+ */
+class CreditCardPaymentMethod extends Payment_Method
+{
+    /**
+     * @inheritdoc
+     */
+    protected static $_sName = 'creditcard';
+
+    /**
+     * @inheritdoc
+     *
+     * @return Transaction
```

Comment:
```suggestion
     * @return CreditCardTransaction
```

If you use that you can omit `use Wirecard\PaymentSdk\Transaction\Transaction` above.

Reply:
👍